### DISCUSSION TITLE
Warp scripts on WarpX

### DISCUSSION THREAD

**Discussion by Byeerendon** [BLAST-WarpX/warpx#4619] [Q&A]
Dear All,

I will your help with the following question:

Can we run Warp scripts on WarpX?  

Best regards,

Bruce

**Comment by ax3l** [comment]
Hi @Byeerendon,

Although most functionality of Warp exists in WarpX now, the scripts of Warp cannot be immediately run in WarpX.

We try to create a lot us domain-specific and detailed input examples though here:
  https://warpx.readthedocs.io/en/latest/usage/examples.html

and have less documented, but plenty benchmark tests here:
  https://github.com/ECP-WarpX/WarpX/tree/development/Examples/Tests

Let us know if you find examples missing!

**Reply by Byeerendon** [reply]
Dear ax3I,

Thanks for the answer and the information of the examples,

Best regards,

Bruce
